deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -249,19 +246,13 @@
     <ClangCompile Include="..\src\shims\mach.c" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
   <PropertyGroup>
-    <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(WINOBJC_SDK_ROOT)\deps\prebuilt\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
+    <PrebuiltRoot Condition="'$(PrebuiltRoot)' == ''">$(WINOBJC_SDK_ROOT)\tools\deps\prebuilt\\</PrebuiltRoot>
+    <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(PrebuiltRoot)\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
   </PropertyGroup>
-  <Target Name="CopyContent" AfterTargets="Build">
+  <Target Name="RemovePackageBinaries" BeforeTargets="Build;Deploy;_CopyFilesMarkedCopyLocal">
     <ItemGroup>
-      <LibraryOutput Include="$(OutDir)\*.lib" />
-      <LibraryOutput Include="$(OutDir)\*.dll" />
-      <LibraryOutput Include="$(OutDir)\*.pdb" />
-      <Headers Include="..\dispatch\*.h" />
-      <Headers Include="..\src\*.h" />
+      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" />
```

Comment:
><ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" /> [](start = 6, length = 63)

What is this trying to do and why?

Reply:
Brian, you and I discussed this a long time ago. The Language package includes libdispatch, and if not ignored it will copy it over the newly compiled version.